src/lighting/lighting-controller.js

```diff
@@ -368,6 +372,9 @@ class LightingController {
         _self.request.time = moment().add(5, 'minutes');
         _self.turnOffLights();
         break;
+      case 'color':
```

Comment:
Is the case block within the _confirmAction_ method necessary? That method is typically used for the conversational commands, which require multiple interactions with Jibo (ex: answering a question that Jibo asks).